src/main/resources/admin/tools/content-type-manager/components/structure/component-option-set-options-option.html

```diff
@@ -69,6 +79,24 @@
           return `${this.path}elements[${index}]`;
         },
 
+        moveOptionUp() {
```

Comment:
There two functions are repeated a lot. Is it possible to create function that is used multiple places / components?